code/game/objects/items/tanks/tanks.dm

```diff
@@ -279,3 +279,36 @@
 
 	else if(integrity < 3)
 		integrity++
+
+/obj/item/tank/attack_hand(mob/living/carbon/human/user)
+	var/celsius_temperature = air_contents.temperature-T0C					//i know i could make the above one public but im too afraid to mess with it
+
+	//recycled code from light bulbs because im too dumb myself
```

Comment:
I smell something that should be a component

Reply:
You'd have to ask the person who wrote the code (who is apparently banned from making changes to the repository), because I wouldn't know how to change this. I only saw the closed pr and figured it was a waste of a good idea to be removed just because the writer was using an alt to bypass a ban.

Reply:
🙄